awx/main/utils/common.py

```diff
@@ -802,7 +802,7 @@ def wrap_args_with_proot(args, cwd, **kwargs):
     for path in sorted(set(show_paths)):
         if not os.path.exists(path):
             continue
-        path = os.path.realpath(path)
+        path = os.path.join(os.path.realpath(path), '')  # add a trailing slash
```

Comment:
Someone's going to pass a file to hide/show paths at some point (if they haven't already)

Reply:
bwrap doesn’t support that

Reply:
Supports it fine.

```
[root@nostromo ~]# bwrap --dev-bind / / --bind /etc/group /etc/even-more-group bash
[root@nostromo ~]# ls -l /etc/even-more-group 
-rw-r--r--. 1 root root 1146 Feb  2 19:53 /etc/even-more-group
```

Reply:
Yea, my batting average isn’t great today.